src/components/imageeditor/imageeditor.js

```diff
@@ -264,7 +264,9 @@ function showImageDownloader(page, imageType) {
         ).then(function () {
             hasChanges = true;
             reload(page);
-        });
+        }).catch(function () {
+            console.debug('No changes were made.');
```

Comment:
```suggestion
            // image downloader closed
```

I don't think it's necessary to log a message